bentoml/transformers.py

```diff
@@ -394,6 +394,21 @@ def _internal_save(
     _check_flax_supported()  # pragma: no cover
```

Comment:
what's the reason for the change from `_save` to `_internal_save`?

Reply:
just for consistency with paddlepaddle